tests/onnx/quantization/common.py

```diff
@@ -95,8 +95,33 @@ def ptq_quantize_model(
     return quantized_model
 
 
+def check_nx_graph(nx_graph: nx.DiGraph, expected_graph: nx.DiGraph):
```

Comment:
Is this a copy-and-paste from the corresponding function in PT code? Extract common code.

Reply:
No, it is different

Reply:
There are already three functions to that name in NNCF test code, which all seem to do one and the same thing. Review all four of the functions, including your own, and extract common code so that, preferrably, one and the same logic is used in all places. If the same logic cannot be used everywhere, give a reason why, name different functions in a different way and describe the mechanism in the docstring for the function.

Reply:
I looked at them. The function very different. It doesn't make any sense to combine them. 
For exapmle, torch version doesn't check edge attributes. Also it removes special attribute 'scope', it works with the specific folder. 
The similar part if checking for nodes attributes, but torch version has special attributes working:
```
        if 'scope' in load_attrs:
            load_attrs.pop('scope')  # TODO: remove
        if attrs != load_attrs:
            assert attrs == load_attrs
```
So, it doesn't make any sense to combine them.

Reply:
The 'scope' parameter is no longer used in the PT testing and this code can be safely removed. None of the torch reference graph files contain this attribute any more, except for the `gnmt_variable.dot` file which is associated with the test that was skipped for the last year or so and corresponds to a pretty much unused and non-working functionality of RNN compression. Remove this torch-specific part of code, along with the `scope=""` parts from the `gnmt_variable.dot` files.

For disabling edge attribute checks, add a `check_edge_attrs: bool = True` parameter into the common `check_nx_graph` function and use it in PT code correspondingly with `check_nx_graph(..., check_edge_attrs=False)`.

Reply:
So, beacuse of that I told you that it takes much time to do that. Because these functions have own specific which I don't know

Reply:
You still have not moved this function to common.

Reply:
Forgot to remove it from ONNX. Now it is done